### PR TITLE
feat(ENG-34): operator dogfooding

### PR DESCRIPTION
**Pull request by ilaif**
This PR Addresses feat(ENG-34): operator dogfooding:
* feat: remove dryRun value 
* feat: add values for deployment annotations 
* feat: add values for deployment env vars 
* feat: add global suffix value to allow creating cluster roles with env name suffixes 
* fix: sqs to only be installed when cloudResources are enabled

### CODE REVIEW

File: charts/velocity-operator/templates/manager-rbac.yaml [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
 kind: ClusterRole
 metadata:
-  name: {{ include "chart.fullname" . }}-manager-role
+  name: {{ include "chart.fullname" . }}-manager-role{{ .Values.globalSuffix | default "" }}
   labels:
   {{- include "chart.labels" . | nindent 4 }}
 rules:
-- apiGroups:
-  - "" # core api group
-  - apps
```
Comment (IamShobe) [inline charts/velocity-operator/templates/manager-rbac.yaml:10]: Will siso approve this?

**Comment by IamShobe** [comment]
Lgtm 💪

**Review by IamShobe — APPROVED** [review]
Lgtm 💪

File: charts/velocity-operator/templates/manager-rbac.yaml (hunk 1 above)
Comment (ilaif) [reply charts/velocity-operator/templates/manager-rbac.yaml:10]: Apparently, we got a lot of problems and a few customers told us it's totally fine. We'll add hardening to that when we get requests for that 🙏🏼

Comment (ilaif) [reply charts/velocity-operator/templates/manager-rbac.yaml:10]: I've added a flag for that

**Pull request merged**